tests/Meilisearch.Tests/IndexTests.cs

```diff
@@ -47,75 +50,89 @@ public async Task IndexCreationWithPrimaryKey()
         public async Task BasicUsageOfIndexMethod()
         {
             var indexUid = "BasicUsageOfIndexMethodTest";
-            var index = this.defaultClient.Index(indexUid);
+            var index = this.client.Index(indexUid);
             index.Uid.Should().Be(indexUid);
             index.PrimaryKey.Should().BeNull();
-            MeilisearchApiError ex = await Assert.ThrowsAsync<MeilisearchApiError>(() => this.defaultClient.GetIndexAsync(indexUid));
-            Assert.Equal("index_not_found", ex.Code);
+            MeilisearchApiError ex = await Assert.ThrowsAsync<MeilisearchApiError>(() => this.client.GetIndexAsync(indexUid));
+            Assert.Equal(ex.Code, "index_not_found");
```

Comment:
To solve the GH Actions:

```suggestion
            Assert.Equal("index_not_found", ex.Code);
```